lib/masamune/tasks/hive_thor.rb

```diff
@@ -39,20 +39,14 @@ class HiveThor < Thor
     method_option :output, :aliases => '-o', :desc => 'Save SQL output to file'
     method_option :delimiter, :desc => 'Hive row format delimiter', :default => "\001"
     method_option :csv, :type => :boolean, :desc => 'Report SQL output in CSV format', :default => false
-    method_option :variables, :aliases => '-D', :type => :hash, :desc => 'Variables to substitute in SQL', :default => {}
+    method_option :variables, :aliases => '-X', :type => :hash, :desc => 'Variables to substitute in SQL', :default => {}
```

Comment:
By itself `-D` is being consumed by thor, with other options it was processed correctly